src/test/components/__snapshots__/Home.test.js.snap

```diff
@@ -334,7 +334,6 @@ exports[`app/Home renders the install screen for the legacy add-on 1`] = `
             To start recording a performance profile in Firefox, first install the
              
             <a
-              className={undefined}
```

Comment:
it looks like that when a prop has an `undefined` value it's not output in snapshots anymore, which seems reasonnable.

Reply:
Looks cleaner.